### ISSUE TITLE
BUG: context.getScope is not a function (with ESLint v9)

### ISSUE THREAD

**Issue by stephen-carter-at-sf** [salesforce/eslint-plugin-lwc#202]
In an attempt to upgrade Salesforce Code Analyzer to ESLint v9, I'm now seeing during testing:
```
Error: The eslint engine encountered an unexpected error thrown from 'ESLint.lintFiles': TypeError: context.getScope is not a function
Occurred while linting /Users/stephen.carter/github/forcedotcom/code-analyzer-core/packages/code-analyzer-eslint-engine/test/test-data/legacyConfigCases/workspace_NoCustomConfig/dummy1.js:2
Rule: "@lwc/lwc/prefer-custom-event"
```

This is a known issue with v9 that needs to be addressed by plugin authors:
https://eslint.org/docs/latest/use/troubleshooting/v9-rule-api-changes

Issue seems to be coming from https://github.com/salesforce/eslint-plugin-lwc/blob/master/lib/util/context.js#L37

**Comment by ravijayaramappa** [comment]
@stephen-carter-at-sf Do you happen to have a quick repro for the issue so we can investigate it?

**Comment by stephen-carter-at-sf** [comment]
Hmmm... this bug might be tricky to reproduce - it may just be the wonky things I'm doing trying to support both flat and legacy configs at the same time. Feel free to disregard this bug - until I can reliably reproduce it without doing wonky things.
